examples/s3express/s3_express_bucket.tf

```diff
@@ -0,0 +1,57 @@
+# Creating the bucket, access controls and lifecycle
+locals {
```

Comment:
Why not have the buckets be created/managed by the module itself? That way people don't have to think about it at all. They just declare if they want s3express or not and voila everything works.